integrations/facebook-pixel/lib/index.js

```diff
@@ -226,6 +267,8 @@ FacebookPixel.prototype.productListViewed = function(track) {
     'ViewContent',
     merge(
       {
+        // Note: `contentIds` here does not go through `getContentIds` since current behavior is
```

Comment:
Not sure if this was an intentional omission but I don't want to change the current behavior either.